games-arcade/savagewheels/metadata.xml

```diff
@@ -1,10 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-	<maintainer type="person">
-		<email>nvinson234@gmail.com</email>
-		<name>Nicholas Vinson</name>
-	</maintainer>
 	<maintainer type="project">
```

Comment:
This package has no other maintainers, so you need to wipe the project as well and send 'up for grabs' mail.